### ISSUE TITLE
gocode autocompletion not working when using // in Sprintf

### ISSUE THREAD

**Issue by norilt** [Microsoft/vscode-go#2240]
If you are using fmt.Sprintf("//%s", #), no autocompletion on postion # is available. However with vim and mdempsky/gocode it is working.

Example Code:
```go
package main

import (
	"fmt"
)

type test struct {
	Test2 string
}

func main() {
	var t test
	fmt.Sprintf("http://%s", ####)
}
```

When you are running gocode in debug mode, you can see, that no request against gocode is made. It looks like vscode is consuming the request.

```
vscode 1.30.2
ms-vscode.Go@0.8.0
go version go1.11.4 linux/amd64
```

**Comment by ramya-rao-a** [comment]
Thanks for reporting this @norilt 

This is happening because there is a logic in place to avoid calling gocode when anything is typed after `//` as it is thought to be a comment.

We need to be a little more smarter there and not consider `//` inside a string as comment

**Comment by ramya-rao-a** [comment]
PRs are welcome to fix this bug

Code Pointers:
- [Build and Debug extension](https://github.com/Microsoft/vscode-go/wiki/Building,-Debugging-and-Sideloading-the-extension-in-Visual-Studio-Code#building-and-debugging-the-extension)
- This is where we exit early when cursor is after a `//`: https://github.com/Microsoft/vscode-go/blob/0.8.0/src/goSuggest.ts#L126
- There is a helper method `isPositionInString` that we can use to check that the position where the `//` is at, is in a string or not.

**Comment by rajdhandus** [comment]
@ramya-rao-a  - can i please work on this issue?

**Comment by oneslash** [comment]
@rajdhandus feel free to fix it

**Comment by mossplix** [comment]
Is someone working on this?

**Comment by oneslash** [comment]
since there was no PR from @rajdhandus, I have done mine #2316

**Comment by ramya-rao-a** [comment]
The latest [beta version](https://github.com/Microsoft/vscode-go/wiki/Use-the-beta-version-of-the-latest-Go-extension) (0.9.3-beta.2) of this extension has the fix for this issue

**Comment by ramya-rao-a** [comment]
This bug fix is out in the latest release(0.10.0) of the Go extension
